packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/User.kt

```diff
@@ -68,6 +68,30 @@ public interface User {
      */
     public val identities: List<UserIdentity>
 
+    /**
+     * Returns the provider type used to log the user in
+     * If a user logs out the authentication provider persists.
+     */
+    public val provider: AuthenticationProvider
+
+    /**
+     * Returns the current access token for the user.
+     * If a user logs out the access token is removed.
+     */
+    public val accessToken: String
+
+    /**
+     * Returns the current refresh token for the user.
+     * If a user logs out the refresh token is removed.
+     */
+    public val refreshToken: String
+
+    /**
+     * Returns a unique identifier for the device the user logged in to.
+     * If a user logs out the device id persists.
```

Comment:
```suggestion
```

I would just remove this sentence here since the device ID doesn't seem connected to the user being logged in or not.